api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java

```diff
@@ -17,21 +22,29 @@
  */
 @Service
 public class ReportingServiceImpl implements ReportingService {
+  private static final Logger logger = Logger.getLogger(ReportingServiceImpl.class.getName());
 
   private final ReportingSnapshotService reportingSnapshotService;
   private final ReportingQueryService reportingQueryService;
   private final ReportingUploadService reportingUploadService;
   private final Provider<WorkbenchConfig> workbenchConfigProvider;
+  private final ReportingVerificationService reportingVerificationService;
+
+  @VisibleForTesting
+  static final List<String> BATCH_UPLOADED_TABLES =
```

Comment:
nit: since use case is mostly to check for presence, a `Set` seems more appropriate to me